### DISCUSSION TITLE
Change the size of the MaterialDesignSwitchToggleButton

### DISCUSSION THREAD

**Discussion by sylvesp** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2943] [Q&A] [answered]
By Default the MaterialDesignSwitchToggleButton has in it a 
`<ViewBox Width="34">`
and that makes the ToggleButton be larger than I need it to be.

I figured that I can put the SwitchToggleButton inside a parent 
```
  <Viewbox Height="20" Width="20">
     <ToggleButton />
  </Viewbox>
```

The question is though: Is there a possibility to make my SwitchToggleButtons smaller without adding an extra container for them?
Something like:
`<ToggleButton  SomehowSetTheWidthOfTheInternalTemplateViewBox="20"/>`  😃 

Thanks for your help in advance!

(Sorry, I am not (yet) a super-duper expert of the WPF intricacies :( )

**Comment by Keboo** [comment]
Hi @sylvesp good question. At present there is not an option (beyond the Viewbox option you listed) for that. However, I did just open up PR #2977 that will let you control the size by setting the Width of the ToggleButton.

**Comment by MichelMichels** [comment, marked as answer]
At moment of writing, the code is `<ViewBox Width="{TemplateBinding Width}" ... />`, effectively binding the ToggleButton.Width to the ViewBox.Width.
